### PR TITLE
store/localstore: fix local region select table in desc order.

### PR DESCRIPTION
**Pull request by coocood**
@shenli @disksing @BusyJay PTAL

### CODE REVIEW

File: executor/builder.go [hunk 1]
```diff
@@ -167,7 +167,7 @@ func (b *executorBuilder) buildTableScan(v *plan.TableScan) Executor {
 		seekHandle:  math.MinInt64,
 	}
 	x := b.buildFilter(e, v.FilterConditions)
-	if v.Desc && !supportDesc {
```
Comment (shenli) [inline executor/builder.go]: Why not consider supportDesc any more?

Comment (coocood) [inline executor/builder.go]: Because our old plain executor `TableScan` and `IndexScan` doesn't support descending order.

Comment (shenli) [inline executor/builder.go]: You mean supportDesc will always be false?

Comment (coocood) [inline executor/builder.go]: `supportDesc` represents if KV layer supports XAPI descending order. 
If we can't use XAPI, this variable should not be used.

**Comment by shenli** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**